storage/engine/mvcc.go

```diff
@@ -1097,15 +1164,17 @@ func MVCCFindSplitKey(engine Engine, raftID int64, key, endKey proto.Key) (proto
 // used after a range is split to recompute stats for each
 // subrange. The start key is always adjusted to avoid counting local
 // keys in the event stats are being recomputed for the first range
-// (i.e. the one with start key == KeyMin).
-func MVCCComputeStats(engine Engine, key, endKey proto.Key) (MVCCStats, error) {
+// (i.e. the one with start key == KeyMin). The elapsedNanos arg
+// specifies the wall time in nanoseconds since the epoch and is used
```

Comment:
@bdarnell renamed to IntentLastUpdateNanos.

@mrtracy time.Duration is actually a pain to work with as you must continually use the Nanoseconds() operator (or more opaquely, cast to int64). As long as you name a variable with a Nanos suffix, I think it's a fine alternative.